### ISSUE TITLE
Update to newer version of GitHub pages?

### ISSUE THREAD

**Issue by karenmajewicz** [geoblacklight/geoblacklight.github.io#128]
When I try to run the site locally, I have some unresolvable errors using the current setup.  There seems to be some incompatibility with older versions of GitHub Pages and newer gems. The main block I get when trying to bundle is `Could not find rdiscount-2.1.8 in any of the sources (Bundler::GemNotFound)`

After a little research, I experimented by updating the Gemfile locally from 
`gem 'github-pages', '48'`
to
`gem 'github-pages', '100'`

This basically seemed to fix it.  

I believe that this is a newish problem from the last month or so. Has anyone else encountered it? I have never managed a GitHub Pages site before. Should we be updating it regularly? The most recent available version is 209.

**Comment by mejackreed** [comment]
What commands are you running locally to run the site?

Prepending `bundle exec` will use the gems specified by the Gemfile and Gemfile.lock , your current "bundle"

**Comment by karenmajewicz** [comment]
Thanks! I did use `bundle exec` to run the site. 

As far as I can tell, the older GitHub pages version uses `rdiscount 2.1.8`. But, when I try to bundle install, it can't find that that version of `rdiscount`. If I try to install the gem directly from rubygems.org, it just hangs.

I got some of the same get_flags errors as in this issue (https://github.com/davidfstr/rdiscount/issues/145), so this may have to do with my XCode version. I will update to version 12.1 and see if that fixes it.
